src/js/components/Notification/Notification.js

```diff
@@ -0,0 +1,70 @@
+import React, { useContext } from 'react';
+import { ThemeContext } from 'styled-components';
+import { defaultProps } from '../../default-props';
+
+import { Layer } from '../Layer';
+import { Box } from '../Box';
+import { Button } from '../Button';
+import { Text } from '../Text';
+import { Paragraph } from '../Paragraph';
+
+const Notification = ({ toast, message, body, status, onClose }) => {
+  const theme = useContext(ThemeContext) || defaultProps.theme;
+
+  const { icon: CloseIcon } = theme.notification.button;
+  const { icon: StatusIcon, color } = theme.notification[status];
+  const iconColor = theme.notification.icon.color;
+
+  let content = (
+    <Box direction="row">
+      <Box
+        {...theme.notification.iconContainer}
+        background={color}
+        justify="center"
+      >
+        <StatusIcon color={iconColor} />
+      </Box>
+      <Box
+        {...theme.notification.textContainer}
+        align="start"
+        direction="row"
+        justify="between"
+        fill
+      >
+        <Box>
+          <Text {...theme.notification.messageText}>{message}</Text>
+          {body && (
+            <Paragraph {...theme.notification.bodyText}>{body}</Paragraph>
+          )}
+        </Box>
+        {onClose && (
+          <Button
+            icon={<CloseIcon color={iconColor} />}
+            onClick={onClose}
+            plain
+          />
+        )}
+      </Box>
+    </Box>
+  );
+
+  if (toast) {
+    content = (
+      <Layer animation="fadeIn" modal={false} onEsc={onClose}>
+        <Box {...theme.notification.toast.container}>{content}</Box>
```

Comment:
Is `role` an attribute we need to consider? 

https://www.w3.org/WAI/WCAG21/quickref/?showtechniques=413#status-messages

https://www.figma.com/file/FGvOd2htjrChVVS2oz73ds/HPE-Notifications?node-id=1382%3A1570